doc/modules/cassandra/examples/CQL/ddm_create_table_with_udf.cql

```diff
@@ -2,7 +2,7 @@ CREATE FUNCTION redact(input text)
    CALLED ON NULL INPUT
    RETURNS text
    LANGUAGE java
-   AS 'return "redacted";
+   AS 'return "redacted";';
```

Comment:
This has been kindly found by @sarma1807 . I'm piggy backing this little fix here since it probably isn't worth a full ticket.

Reply:
Thank you very much for the fix @adelapena